hoodie-common/src/main/java/com/uber/hoodie/common/util/HoodieTimer.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2018 Uber Technologies, Inc. (hoodie-dev-group@uber.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *          http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.uber.hoodie.common.util;
+
+import com.uber.hoodie.exception.HoodieException;
+import java.util.ArrayDeque;
+import java.util.Deque;
+
+/**
+ * Timing utility to help keep track of execution times of code blocks. This class helps to allow multiple
+ * timers started at the same time and automatically returns the execution time in the order in which the
+ * timers are stopped.
+ */
+public class HoodieTimer {
+
+  // Ordered stack of TimeInfo's to make sure stopping the timer returns the correct elapsed time
+  Deque<TimeInfo> timeInfoDeque = new ArrayDeque<>();
```

Comment:
Not sure if the comment got lost. But I was asking if we can use a StopWatch from apache commons or some standard class here..

Reply:
I didn't get lost, I left a comment earlier.The comments get collapsed so you might have missed seeing it. Pasting my earlier comment here again : 

```
So there are 2 advantages of doing this : 1) Centralized place for all timers in case in the future a refactor is done for timer class used. 2) Using just one timer instance you can track multiple code blocks instead of instantiating everywhere..
```

Reply:
Okay. not fully onboard :). but ok , lets go ahead